onnx/mapping.py

```diff
@@ -36,6 +36,7 @@
     TensorProto.INT64: TensorProto.INT64,
     TensorProto.BOOL: TensorProto.INT32,
     TensorProto.FLOAT16: TensorProto.UINT16,
+    TensorProto.BFLOAT16: TensorProto.UINT16,
```

Comment:
Actually now this whole `TENSOR_TYPE_TO_STORAGE_TENSOR_TYPE` mapping seems unnecessary to me and I don't remember why I added it :-). I will remove this map in a follow up PR.

Reply:
I thought an initial type is mapped to storage type, which is then mapped to name of the field

Reply:
yeah but IIRC we never really used storage type directly (it's always followed by using it to look up in the STORAGE_TENSOR_TYPE_TO_FIELD map).